libraries/bot-integration-core/src/test/java/com/microsoft/bot/integration/CloudAdapterTests.java

```diff
@@ -77,36 +81,6 @@ public void basicMessageActivity() {
         Mockito.verify(botMock, Mockito.times(1)).onTurn(Mockito.any(TurnContext.class));
     }
 
-    @Test
```

Comment:
Just to know. Why was this removed?

Reply:
All is explained [here](https://southies.slack.com/archives/C01HGUQ4UDU/p1626188521220600).

But, summarizing:
The behavior between Java and C# differs here, C# is mocking a `HttpRequest` which is present in [Conversation.cs](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Connector/Conversations.cs) that Java handle it using an external service `RestClient` used by [RestConversation.java](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-connector/src/main/java/com/microsoft/bot/connector/rest/RestConversations.java).

As C# is mocking a call that Java doesn't have because it's handled by the external service, the **test is useless** in Java as it's not validating anything from the `CloudAdapter` migration.

I had some doubts if we should or not remove this, because also C# is useless, we will confirm if it's necessary during the JaCoCo but also during the validation of the samples.